Nighttime-Operations/Auxiliary-Telescope/AT-On-Sky/CWFS.rst

```diff
@@ -4,11 +4,92 @@
 
 .. _AT-On-sky-CWFS:
 
-#######################
-CWFS
-#######################
+#################################
+Focus, center and absorb offsets
+#################################
+
+.. _Beginning_of_the_night-the-Telescope-Overview:
+
+Overview
+========
+
+This procedure must be followed by the observers before the beginning of dark time, prior to starting observations. 
```

Comment:
Consider:
... prior to starting science or engineering activities.

Reply:
👍